NHLBI_BioData_Catalyst/R/5_LongitudinalData.ipynb

```diff
@@ -87,13 +93,11 @@
    "source": [
     "Sys.setenv(TAR = \"/bin/tar\")\n",
     "options(unzip = \"internal\")\n",
-    "install.packages(\"https://cran.r-project.org/src/contrib/Archive/devtools/devtools_1.13.6.tar.gz\", repos=NULL, type=\"source\")\n",
-    "install.packages(\"https://cran.r-project.org/src/contrib/R6_2.5.1.tar.gz\", repos=NULL, type=\"source\")\n",
-    "install.packages(\"https://cran.r-project.org/src/contrib/hash_2.2.6.1.tar.gz\", repos=NULL, type=\"source\")\n",
-    "install.packages(c(\"urltools\"),repos = \"http://cran.us.r-project.org\")\n",
     "devtools::install_github(\"hms-dbmi/pic-sure-r-client\", force=T)\n",
     "devtools::install_github(\"hms-dbmi/pic-sure-r-adapter-hpds\", force=T)\n",
-    "devtools::install_github(\"hms-dbmi/pic-sure-biodatacatalyst-r-adapter-hpds\", force=T)"
+    "devtools::install_github(\"hms-dbmi/pic-sure-biodatacatalyst-r-adapter-hpds\", force=T)\n",
+    "library(hpds)\n",
+    "library(ggplot2)"
```

Comment:
Need to include library(stringr) to have the notebook run successfully in Terra. Will check SB when functional again.

Reply:
Resolved in [27cd4656b99c1408a316c0f7120b3ac4d4f6d169](https://github.com/hms-dbmi/Access-to-Data-using-PIC-SURE-API/commit/27cd4656b99c1408a316c0f7120b3ac4d4f6d169)